Makefile

```diff
@@ -5,6 +5,9 @@ PYTHON ?= python
 DESTDIR ?=
 SIGN ?=
 PREFIX ?= /usr/local
+STAGING_SERVER ?= iojs-www
+
+OSTYPE := $(shell uname -s | tr '[:upper:]' '[:lower:]')
```

Comment:
It's probably an academic issue but when libuv was still Makefile-based, we used `tr '[A-Z]' '[a-z]'` because `tr '[:upper:]' '[:lower:]'` might be affected by the locale.